src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -71,24 +68,6 @@ class HaConfigDashboard extends LitElement {
             </ha-card>
           `
         : ""}
-      ${CONF_HAPPENING && !localStorage.dismissConf2020
-        ? html`
-            <ha-card class="conf-card">
-              <a
-                target="_blank"
-                href="https://www.home-assistant.io/conference"
-                rel="noopener noreferrer"
-              >
-                <img src="/static/images/conference.png" />
```

Comment:
I'm not familiar with the front end structure, but shouldn't this image also have been removed as part of this pr?

Reply:
[`5cc49bb` (#7731)](https://github.com/home-assistant/frontend/pull/7731/commits/5cc49bbd10cb6b9ddb120fe1ade7625633c7d0ae)